lib/rspec/rails/active_record.rb

```diff
@@ -11,7 +11,7 @@ def self.initialize_activerecord_configuration(config)
             ::RSpec::Mocks.configuration.when_declaring_verifying_double do |possible_model|
               target = possible_model.target
 
-              if target.respond_to?(:define_attribute_methods) && ActiveRecord::Base != target
+              if target.respond_to?(:define_attribute_methods) && ActiveRecord::Base != target && !target.abstract_class
```

Comment:
Why `abstract_class` instead of `abstract_class?`

http://api.rubyonrails.org/classes/ActiveRecord/Inheritance/ClassMethods.html#method-i-abstract_class-3F

Also, I don't think that you can assume `target` will respond to `abstract_class` just because it responds to `define_attribute_methods`.  Consider that ActiveModel provides` define_attribute_methods` as well:

http://api.rubyonrails.org/classes/ActiveModel/AttributeMethods/ClassMethods.html#method-i-define_attribute_methods

So this needs a `respond_to?(:abstract_class)` check.

On a side note, I was hoping that `ActiveRecord::Base.abstract_class?` would return `true` (it is abstract after all...) so that we could do away with the explicit `ActiveRecord::Base != target` check but sadly, it doesn't :(.